api/ombpdf/management/commands/scrape_memoranda.py

```diff
@@ -42,13 +43,19 @@ def parse_pdf(policy: Policy, url: Url) -> bool:
         pdf = download_with_progress(url)
         pdf.name = basename(urlparse(url).path)     # this used by from_file
         doc = OMBDocument.from_file(pdf)
+        with reversion.create_revision():
+            policy.workflow_phase = WorkflowPhases.cleanup.name
+            policy.save()
```

Comment:
Should this probably happen after the `to_db` line, in case something goes wrong there?

Reply:
Yup, it definitely should.